converter/internal/staticconvert/validate.go

```diff
@@ -98,11 +98,12 @@ func validateMetrics(metricsConfig metrics.Config, grpcListenPort int) diag.Diag
 func validateIntegrations(integrationsConfig config.VersionedIntegrations) diag.Diagnostics {
 	var diags diag.Diagnostics
 
-	if len(integrationsConfig.EnabledIntegrations()) == 0 {
-		return diags
-	}
-
 	for _, integration := range integrationsConfig.ConfigV1.Integrations {
+		if !integration.Common.Enabled {
+			diags.AddWithDetail(diag.SeverityLevelError, fmt.Sprintf("unsupported disabled integration %s.", integration.Name()), "")
```

Comment:
why not just "Add" instead of "AddWithDetail" since you set "" for the details?
`func (ds *Diagnostics) Add(severity Severity, message string)`

Reply:
my main reason is that I was not being smart. updated, good catch